processing/src/main/java/org/apache/carbondata/processing/loading/CarbonDataLoadConfiguration.java

```diff
@@ -460,4 +464,20 @@ public String getColumnCompressor() {
   public void setColumnCompressor(String columnCompressor) {
     this.columnCompressor = columnCompressor;
   }
+
+  public String getAppName() {
+    return appName;
+  }
+
+  public void setAppName(String appName) {
+    this.appName = appName;
+  }
+
+  public String getVersion() {
+    return version;
+  }
+
+  public void setVersion(String version) {
```

Comment:
Is this carbon jar version? or application version? Application version should be given in the writtenBy

Reply:
it is carbon version